.github/workflows/codeql.yml

```diff
@@ -1,37 +1,93 @@
-name: 'CodeQL'
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
 
 on:
+  push:
+    branches: [ "main" ]
   pull_request:
-    types: [opened, synchronize]
+    branches: [ "main" ]
+  schedule:
+    - cron: '25 15 * * 2'
```

Comment:
why do we need this job scheduled if it will already run on every push to main?  Is there something that this can catch that will change between pushes?

Reply:
We can definitely take this out, I just want to make sure the codeql works with the default config before adjusting to fit our specific needs.